src/utils/SdkVersionMiddleware.ts

```diff
@@ -21,17 +22,34 @@ export class SdkVersionMiddleware implements Middleware {
    * @private
    * A member to hold next middleware in the middleware chain
    */
-  private nextMiddleware: Middleware;
+  private _nextMiddleware: Middleware;
+  private _component?: MgtBaseComponent;
+
+  constructor(component?: MgtBaseComponent) {
+    this._component = component;
+  }
 
   // tslint:disable-next-line: completed-docs
   public async execute(context: Context): Promise<void> {
     try {
-      let sdkVersionValue: string = `mgt/${PACKAGE_VERSION}`;
+      const headerParts: string[] = [];
+
+      // Component version
+      if (this._component) {
+        const componentVersion: string = `${this._component.tagName}/${this._component.version}`;
```

Comment:
version should be PACKAGE_VERSION - all components move up the version at the same time

Reply:
So we don't have any interest in versioning controls individually? We aren't guaranteed to need to rev the version for controls we don't touch between releases.  

Also, we are already sending the PACKAGE_VERSION value in the header. Why duplicate it?

Reply:
Followed up with you directly. Switching to PACKAGE_VERSION